build/pack-docs.ps1

```diff
@@ -30,7 +30,7 @@ if ("$Env:BUILD_RELEASETYPE" -eq "release") {
 # easily inject the right base image into the FROM line. In doing so,
 # the build context should include the build_docs.py script that we need.
 $dockerfile = @"
-FROM ${Env:DOCKER_PREFIX}iqsharp-base:${Env:BUILD_BUILDNUMBER}
+FROM ${Env:DOCKER_PREFIX}iqsharp-base:${Env:SEMVER_VERSION}
```

Comment:
I think the problem you'll run into here is that `Env:SEMVER_VERSION` is not defined in `set-env.ps1`; that's a variable that only exists in the end-to-end QDK pipeline.

Probably instead you'd want to make the image version a parameter to `pack-docs.ps1`. Then the CI build here can pass `Env:BUILD_BUILDNUMBER`, and the QDK pipeline can pass the SemVer version.

Reply:
Makes sense. I'll start that approach as a separate PR.